provider/marathon.go

```diff
@@ -179,7 +179,15 @@ func (provider *Marathon) getLabel(application marathon.Application, label strin
 	return "", errors.New("Label not found:" + label)
 }
 
-func (provider *Marathon) getPort(task marathon.Task) string {
+func (provider *Marathon) getPort(task marathon.Task, applications []marathon.Application) string {
+	application, errApp := getApplication(task, applications)
+	if errApp != nil {
+		log.Errorf("Unable to get marathon application from task %s", task.AppID)
+		return ""
+	}
+	if label, err := provider.getLabel(application, "traefik.port"); err == nil {
+		return label
```

Comment:
Yeah, that makes good sense.

Would you prefer the label be called "traefik.portIndex" also?